assets/src/app/ato/Documents/Fedramp.tsx

```diff
@@ -12,7 +12,7 @@ import * as Api from '@app/lib/api'
 interface FedRAMPsState {
     rows: IRow[];
 }
-class ATOFedRAMPs extends React.Component<any, FedRAMPsState> {
+class ComponentFedRAMPTemplates extends React.Component<any, FedRAMPsState> {
```

Comment:
Seem to recall PatternFly/react using this naming scheme... but cannot cite sources. Am I going crazy? Can easily revert this change.

Reply:
I didn't noticed many components starting with component keyword.

Nevertheless, no worries, the name can remain what it is.